compile-handlebars-templates.sh

```diff
@@ -4,3 +4,5 @@
 export PATH=./node_modules/.bin/:$PATH
 
 handlebars -n OCA.Talk.Views.Templates js/views/templates/ -f js/views/templates.js
+
+echo "OK"
```

Comment:
Not needed I think, but OK.

Reply:
Yeah I added it, because the handlebar does not produce any output and I was quite confused if it ran or not.